app/models/environment_variable.rb

```diff
@@ -2,4 +2,5 @@ class EnvironmentVariable < Parameter
   belongs_to :container, :foreign_key => :reference_id, :inverse_of => :environment_variables
   audited :except => [:priority], :associated_with => :container, :allow_mass_assignment => true
   validates :name, :uniqueness => { :scope => :reference_id }
+  validates_presence_of :name
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.